dnf5.spec

```
@@ -2,6 +2,8 @@
 %global project_version_minor 1
 %global project_version_patch 1
 
+%bcond_with obsolete_dnf
```

This seems unnecessarily convoluted. How about changing this to

```suggestion
%bcond obsolete_dnf %[0%{?fedora} >= 41 || 0%{?rhel} >= 11]
```

and using simple `%if %{with obsolete_dnf}` and `%if %{without obsolete_dnf}` conditionals everywhere?

---

I like it.